frontend/pages/model/[modelId]/access-request/new.tsx

```diff
@@ -108,8 +108,8 @@ export default function NewAccessRequest() {
       <Title text='Access Request' />
       {isLoading && <Loading />}
       {!isLoading && (
-        <Container maxWidth='md'>
-          <Card sx={{ mx: 'auto', my: 4, p: 4 }}>
+        <Container maxWidth='lg'>
+          <Card sx={{ mx: 'auto', my: 4, p: 5 }}>
```

Comment:
For consistency I'd vote for keeping the padding to 4 as we use that for most of our cards (at least the cards that hold the main content of a page)